coderd/notifications/notifier.go

```diff
@@ -223,6 +229,18 @@ func (n *notifier) prepare(ctx context.Context, msg database.AcquireNotification
 		return nil, xerrors.Errorf("failed to resolve handler %q", msg.Method)
 	}
 
+	logoURL, err := n.store.GetLogoURL(ctx)
+	if err != nil && !errors.Is(err, sql.ErrNoRows) {
+		n.log.Error(ctx, "failed fetching logo url", slog.Error(err))
+	}
+
+	if logoURL == "" {
+		//nolint:ineffassign // define to default value if unable to fetch one from db
```

Comment:
Is this truly an `ineffassign`, I would assume the assign to labels below takes care of it?